pallets/xcmp-queue/src/lib.rs

```diff
@@ -198,7 +205,7 @@ pub mod pallet {
 		/// message sending may recommence after it has been suspended.
 		///
 		/// - `origin`: Must pass `Root`.
-		/// - `new`: Desired value for `QueueConfigData.resume_threshold`                                
+		/// - `new`: Desired value for `QueueConfigData.resume_threshold`
 		#[pallet::weight(10_000_000 as Weight + T::DbWeight::get().reads_writes(1, 1))]
```

Comment:
These weights need to be updated once you have the benchmarked weight files.

Reply:
Ah yes - I see now that's how the WeighInfos get called.